### PR TITLE
zeeshaan fixes to focus area pages

### PR DESCRIPTION
**Pull request by ZeeshaanMaudar**
- Fix url i.e add /focus/ to url
- Show explore button on consolidated again
- Fix live focus area page: 
    - when all data is available
    - only one footnote available for National Treemap
   - both footnote available for National Treemap
   - Show only notices when data is empty array, and total is nulll
   - Show notices and total when data is empty array and total is 0.

### CODE REVIEW

File: packages/webapp/src/views/ConsolidatedTreemap/index.jsx [hunk 1]
```diff
@@ -27,7 +27,6 @@ const Markup = ({ items, initialSelected }) => (
       disabled: "2019-20",
     }}
     anchor="consolidated-treemap"
-    isConsolidatedChart
```
Comment (jbothma) [inline packages/webapp/src/views/ConsolidatedTreemap/index.jsx]: this isn't cool - you're removing the argument saying that this is the consolidated chart, from a component called ConsolidatedTreemap...

Comment (jbothma) [reply packages/webapp/src/views/ConsolidatedTreemap/index.jsx]: Could you suggest a more appropriate solution? Is this to allow the button to show?

Comment (ZeeshaanMaudar) [reply packages/webapp/src/views/ConsolidatedTreemap/index.jsx]: I've removed the logic that takes this boolean now, so that it doesn't look redundant

**Comment by jbothma** [comment]
`http://localhost:4000/2019-20/focus/community-development` still doesn't work for me?

**Comment by ZeeshaanMaudar** [comment]
> `http://localhost:4000/2019-20/focus/community-development` still doesn't work for me?

Weird. Mine is. Even this one: `http://localhost:3000/2019-20/focus/general-public-services`

File: packages/webapp/src/adaptors/focus-areas-preview/Routing.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import {
 const passRouteValues = ({ match }) => createElement(DataLoader, match.params)
 
 const routeConfig = {
-  path: "/:year/:department",
+  path: "/:year/focus/:department",
```
Comment (schalkventer) [inline packages/webapp/src/adaptors/focus-areas-preview/Routing.js]: Perhaps make this something more semantic? (Make note for refactor)

**Pull request merged**